kgforge/core/archetypes/store.py

```diff
@@ -56,7 +56,8 @@
     "prefix",
     "graph",
     "distinct",
-    "in"
+    "in",
+    "as"
```

Comment:
Any other you think you may need? (just taking the chance to add them now)

Reply:
None yet, but I can always add many in preparation for the future, using the[ grammar](https://www.w3.org/TR/rdf-sparql-query/#grammar)